window/src/os/wayland/state.rs

```diff
@@ -82,6 +84,14 @@ impl WaylandState {
         let subcompositor =
             SubcompositorState::bind(compositor.wl_compositor().clone(), globals, qh)?;
 
+        let blur_manager: Result<OrgKdeKwinBlurManager, BindError> =
+            globals.bind(qh, 1..=1, GlobalData);
+
+        let blur_manager = match blur_manager {
+            Ok(manager) => Some(manager),
+            Err(_) => None,
+        };
+
```

Comment:
Can simplify this, I think:

```suggestion
        let blur_manager: Result<OrgKdeKwinBlurManager, BindError> =
            globals.bind(qh, 1..=1, GlobalData).ok();
```